### ISSUE TITLE
86Box unable to boot 320kb Compaq DOS 1.12g image.

### ISSUE THREAD

**Issue by mph2016** [86Box/86Box#902]
**Describe the bug**
Not sure if it's a bug or a bad image; but compaq dos 1.1 won't boot.

**To Reproduce**
1. Download the DOS from archive.org
2. Create a default PC. Alternately set 86Box up as 8088/Compaq portable
3. Insert the 320kb image into the slot for the 'a' drive
4. Cool boot the machine. 
5. Watch as the screen freaks out and then eventually ends up looking similar to the snapshot below.

**Expected behavior**
I expected a normal boot-up process ending with a dos prompt.

**Screenshots**
![20200705-233821-150](https://user-images.githubusercontent.com/17302103/86544898-31bdf480-bed7-11ea-880b-734cd7d3d49b.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 v1909 x64
 - 86Box version: v2.10, build 2424
 - Build type: regular build.

**Additional context**
I found this DOS when reading the comments to a story about gwbasic at the os2museum: 
http://www.os2museum.com/wp/gw-basic-source-notes/#comments  
I followed the link mentioned in the comments to this internet archive copy: 
https://archive.org/details/compaq-dos-1.12g  

I suspect that either 86Box can't boot 320kb images, or that the image on archive.org is corrupted/unbootable. 

I don't know which.

**Comment by mph2016** [comment]
I tried with a couple of DOS images I have. 
One with an IMD extension also wouldn't boot. 
One 320kb dos 1.1 image did boot, so I think that rules out it being a disk capacity problem. 
Working DOS screenshot: 
![20200706-000310-186](https://user-images.githubusercontent.com/17302103/86545154-692da080-bed9-11ea-9ecf-9be1602058b5.png)

Meanwhile, I checked the extention on the compaq DOS: it's also img. 
When put the floppy in the 'b' drive I was able to read it from the working DOS:  
![86BoxScreenCapture-5July2020-03](https://user-images.githubusercontent.com/17302103/86545243-fb35a900-bed9-11ea-8a50-5de361c2e610.png)
